db/migrate/20180323134506_update_view_requests.rb

```diff
@@ -2,6 +2,7 @@
 
 # The requests view had been updated to include request purpose, which is now an enum.
 # This adds the previously untracked request_purpose column to the view in a compatible manner.
+# rubocop:disable Layout/LineLength
```

Comment:
Could be declared at the end of the `class` line?

Reply:
Actually no. It appears that the inline declarations only work for cops which specifically concern the class/method they are inline for. It isn't a way of simply scoping a cop to a class/method.